unumpy/_multimethods.py

```diff
@@ -1872,7 +1872,36 @@ def rot90(m, k=1, axes=(0, 1)):
     return (m,)
 
 
-@create_numpy(_self_argreplacer)
+def _unwrap_default(p, discont=3.141592653589793, axis=-1):
+    nd = ndim(p)
+
+    dd = diff(p, axis=axis)
+
+    slice0 = [slice(None)] * nd
+    slice0[axis] = 0
```

Comment:
You can use `slice(0, 1)` to grab the 0 value without also collapsing the dimension. That will avoid the need to use `expand_dims` later.

Reply:
Good idea, that way we avoid an unnecessary multimethod call.